data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -74,6 +74,25 @@
 ]
 TILES_ROUND_NUM_DECIMALS = 2
 
+# Controlling Tile user experience columns
```

Comment:
I made the choice to move all of this into constants, rather than field names. I think this makes more sense because none of this information is used in scoring and I thought it was clearer this way. Happy to revert if you think the field names belong better with other field names :)

Reply:
I think having the constants (like the threshold count, fips codes, etc) here makes a lot of sense. I'm torn on moving the field names here. That feels somewhat unintuitive since all the other field names are there. But I think it's fine for now? We're clearly planning some big moves around Fields and Models soon so can reconsider all this then.